less/about/siteDetails.less

```diff
@@ -1,16 +1,10 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
 @import "./itemList.less";
 @import "./common.less";
 
-strong, div, span, li, em, p, a {
-  font-weight: 400;
-  -webkit-font-smoothing: antialiased;
-}
-
-body {
-  background: @veryLightGray;
```

Comment:
Deprecated with https://github.com/brave/browser-laptop/pull/10881/files#diff-b79e96c438a5b97dc31f04f68d4817a7R22 on common.less. As common.less is required on all of the about js files, it should not cause a regression.